Rethinking/end-of-chapter-practice-problems/ch-10.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<p>nitpicking: Notice that instead of `fill_between` you can use `az.plot_hpd` https://arviz-devs.github.io/arviz/examples/plot_hpd.html</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
You're right, will use that for next chapters. Just learnt about it in your book ;) Very useful